internal/cluster/refresh_status_preprocessor.go

```diff
@@ -99,6 +123,16 @@ func (r *refreshPreprocessor) preprocess(ctx context.Context, c *clusterPreproce
 	for _, validationResults := range validationsOutput {
 		sortByValidationResultID(validationResults)
 	}
+
+	// Now that all validations have been executed, skip the validations that we wish to ignore.
+	for _, currentResult := range validationsOutput {
```

Comment:
cant we do it on validationsOutput creation? While going through results?

Reply:
This block is here for a reason and in attempting to resolve this comment I am quite sure that it should be here.

The issue is that we are not applying the ignore at the very last moment that anything could alter a condition and in testing, I have just demonstrated that this concern is justified by implementing your suggestion.

So, in the lack of any further suggestions, I opt to leave this code block here. Open to reviewing the block itself to simplify, but not to removing it.

Reply:
I actually though about setting status to passed and then no need to set stateMachineInput[string(v.ID)] = true ?

Reply:
Ah, this was discussed with Ori and I agree with his take on this that it is better to preserve the content and actual state of the validation. 

So we use the statemachine conditions. 

@ori-amizur what do you think?